components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtilsTest.java

```diff
@@ -20,24 +20,38 @@
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
+import org.wso2.carbon.identity.application.authentication.framework.ApplicationAuthenticator;
 import org.wso2.carbon.identity.application.authentication.framework.config.ConfigurationFacade;
 import org.wso2.carbon.identity.application.authentication.framework.handler.request.impl.PostAuthnMissingClaimHandler;
+import org.wso2.carbon.identity.application.authentication.framework.internal.FrameworkServiceComponent;
+import org.wso2.carbon.identity.common.testng.WithCarbonHome;
+import org.wso2.carbon.identity.testutil.powermock.PowerMockIdentityBaseTest;
 
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
+import java.io.UnsupportedEncodingException;
+
+import static org.mockito.Mockito.mock;
 import static org.powermock.api.mockito.PowerMockito.mockStatic;
 import static org.powermock.api.mockito.PowerMockito.when;
 import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
 
-@PrepareForTest({ConfigurationFacade.class})
-public class FrameworkUtilsTest {
+@WithCarbonHome
+@PrepareForTest({ConfigurationFacade.class, FrameworkServiceComponent.class})
+public class FrameworkUtilsTest extends PowerMockIdentityBaseTest {
 
     @Mock
     ConfigurationFacade mockedConfigurationFacade;
 
+
```

Comment:
Unnecessary new line

Reply:
Already fixed with commit baf49d4b0e85432467ae5848dcf443c0032ce5c9 in https://github.com/wso2/carbon-identity-framework/pull/3509 PR